hosting/src/global/App/NavBar/NavBar.html

```diff
@@ -172,6 +172,11 @@
           sm:inset-auto
         "
       >
+        <!-- NEW CODE STARTS HERE -->
+        <select v-model="$i18n.locale" class="appearance-none p-2 mr-2">
```

Comment:
You need to run `yarn fix`

This also suggests you might be missing a VSCode extension, because this should have happened automatically when you saved the file.